stories/Input.story.jsx

```diff
@@ -23,14 +23,14 @@ storiesOf("Input", module)
     ))
   )
   .add(
-    "optional",
+    "default value",
```

Comment:
I think better to add also optional example without default value

Reply:
optional doesn't do anything afaik. It isn't even a prop on the input

Reply:
And the "basic" story, shows an input without any default values.

Reply:
anyway, better to have example with absent default value